scripts/apps/desks/directives/DeskeditBasic.js

```diff
@@ -93,6 +95,12 @@ export function DeskeditBasic(gettext, desks, WizardHandler, metadata, $filter,
                 }
             };
 
+            scope.getNoPublishOnAuthoringDesk = function(deskType) {
```

Comment:
It would be nice to leave a comment. I'm not familiar with this functionality (but would like to be) and I have no idea what "NoPublishOnAuthoringDesk" means.

Additionally, some points:
- `config.features` is always defined in the [default app configuration](https://github.com/superdesk/superdesk-client-core/blob/master/webpack.config.js#L225) so you don't need to check
- `config.features.noPublishOnAuthoringDesk` should be enough, no need to use the `in` operator.
- Because this function is getting the value of a feature and not an item, it might be a bit confusing it to prefix it with _get_

My recommendation would be to make this:
```js
// Checks if the given deskType can publish on the authoring desk.
scope.canPublishOnAuthoring = (deskType) =>
    deskType === 'authoring' && config.features.noPublishOnAuthoringDesk;
```

Reply:
There was a mistake in my code snippet, I have edited the above comment.

Reply:
I think config.feature can be empty, it is just your assumption that it is already prsent.

Reply:
I have already linked to the code where `config.features` is defined, illustrating that it is never empty. I don't understand why you say it's an assumption.

Reply:
Here it is again, in case you've missed it:
https://github.com/superdesk/superdesk-client-core/blob/master/webpack.config.js#L225

It gets assigned here, on app init:
https://github.com/superdesk/superdesk-client-core/blob/0d233de155b60ccc029dd7de480e3e28e7c49a23/scripts/index.js#L10

Reply:
Yes, I missed that link on the first time